### PR TITLE
chore: update logs

### PR DESCRIPTION
**Pull request by asafMasa**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

Update mc-utils and mc-priority-queue packages for better logs

### CODE REVIEW

File: config/default.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   },
   "telemetry": {
     "logger": {
-      "level": "info",
+      "level": "debug",
```
Comment (asafMasa) [inline config/default.json:10]: will be reverted

File: src/clients/callbackClient.ts [hunk 2]
```diff
@@ -7,11 +7,11 @@ import { ICallbackData, ICallbackExportData, IConfig } from '../common/interface
 @singleton()
 export class CallbackClient extends HttpClient {
   public constructor(@inject(SERVICES.LOGGER) logger: Logger, @inject(SERVICES.CONFIG) private readonly config: IConfig) {
-    super(logger, '', 'requestCallback', config.get<IHttpRetryConfig>('httpRetry'));
+    super(logger, '', 'requestCallback', config.get<IHttpRetryConfig>('httpRetry'), false);
```
Comment (ronenkapelian) [inline src/clients/callbackClient.ts]: should it be const and configurale params?

Comment (asafMasa) [reply src/clients/callbackClient.ts]: fixed as requested

**Review by ronenkapelian — APPROVED** [review]
:1st_place_medal:

**Pull request merged**